django-stubs/contrib/admin/models.pyi

```diff
@@ -21,6 +22,15 @@ class LogEntryManager(models.Manager[LogEntry]):
         action_flag: int,
         change_message: Any = ...,
     ) -> LogEntry: ...
+    def log_actions(
+        self,
+        user_id: int,
+        queryset: QuerySet[Model],
+        action_flag: int,
+        change_message: str | list[Any] = "",
+        *,
+        single_object: bool = False,
```

Comment:
Maybe this should be an `@overload`? When `single_object: Literal[True]` it will return `-> LogEntry`, otherwise `-> list[LogEntry]`.

Reply:
Thank you for the review! This approach makes it clearer.